bigtable/cmd/emulator/cbtemulator.go

```diff
@@ -37,11 +38,18 @@ const (
 
 func main() {
 	grpc.EnableTracing = false
+	flag.Var(&emulatorInterceptor.LatencyTargets, "inject-latency", "Introduce gRPC latency for testing like \"ReadRows:p99.9:100ms\"")
```

Comment:
I think the docs here need a bit more detail, it's not obvious to me looking at this what the example you specify will actually do in terms of emulator behavior.